README.md

```diff
@@ -8,3 +8,4 @@ ACS UPB Mobile started as a B.Sc. thesis project. Subsequently, more papers were
 | [Design and implementation of a cross-platform mobile application that facilitates student collaboration](Design%20and%20implementation%20of%20a%20cross-platform%20mobile%20application%20that%20facilitates%20student%20collaboration) | [Ioana Alexandru](https://github.com/IoanaAlexandru) | _June 2020_ | The university is a complex environment with many creative people having as a disadvantage the fact that it sometimes develops chaotically. This development is visible in the vast array of software applications meant to assist students. Over time in our university, many such platforms were developed, including the university website, the faculty website, websites of different student organizations, wikis created by professors or departments, and the official course platform. This variety turned into an information overload for students who are expected to be aware of everything that is posted on these websites, to discern between relevant information and spam, while also busy attending courses and completing assignments. <br/>We believe that students themselves can address this issue by using the power of the community to flag and organize important information from these websites, disseminate it to relevant colleagues and help each other get through the labyrinth of information existing in the university. <br/>With this goal in mind, this paper describes a collaborative app that allows students to better organize their time and schedule by collectively identifying the most useful information from the university’s sources and intelligently sharing it with their peers. |
+| [University schedule personalization andautomatization for productivity](University%20schedule%20personalization%20andautomatization%20for%20productivity) | [Bogdan Iuga](https://github.com/iugabogdan98) | _July 2021_ | This paper presents the implementation and integration of a module dedicated to improving the way of handling university activities inside an already existing application. <br/>The main functionalities implemented were the addition of separated events for university tasks such as homework and projects, the possibility of choosing goals for them, and the creation of visual elements in order to help the user find relevant information faster. <br/>The application being already used by the students of the Faculty of Automatic Control and Computer Science, University POLITEHNICA of Bucharest, there were analyzed the administrative structures inside the faculty and also other applications that have the same functionalities like the ones proposed in this paper, in order to satisfy the needs of the students. <br/>The theme was analyzed from different points of view, such as the used architecture. the user interface, the technologies used. <br/>The development conventions of the application were followed so the result works alongside the other modules and can be delivered to the users. |
```

Comment:
Missing space in title, here and in the directory and PDF name as well ("and automatization")